src/pathfinder.js

```diff
@@ -81,8 +82,21 @@ class Pathfinder {
     return cheapestPath
   }
 
+  // params -
+  //   sourceLedger
+  //   destinationLedger
+  //   destinationAmount
+  //   destinationAccount (optional)
```

Comment:
Oh, please can we not do this? The ripple-lib code base had a lot of this pattern and it turned into a real mess.